### PR TITLE
chore(tfhe): remove last remaining modular_std_dev

### PR DESCRIPTION
**Pull request by IceTDrinker**
- some places were not updated, remove the last non modular std_dev
- the ones to dump parameters are modular so are kept

### CODE REVIEW

**Review by mayeul-zama — APPROVED** [review]

**Comment by IceTDrinker** [comment]
@mayeul-zama one of the tests was failing, I just pushed the fix, but only after you approved once 😅

**Pull request merged**
